core/src/OpenMPTarget/Kokkos_OpenMPTarget_Instance.cpp

```diff
@@ -77,7 +77,16 @@ void OpenMPTargetInternal::impl_finalize() {
     Kokkos::kokkos_free<Kokkos::Experimental::OpenMPTargetSpace>(
         space.m_uniquetoken_ptr);
 }
-void OpenMPTargetInternal::impl_initialize() { m_is_initialized = true; }
+void OpenMPTargetInternal::impl_initialize() {
+  m_is_initialized = true;
+
+  // FIXME_OPENMPTARGET:  Only fix the number of teams for NVIDIA architectures.
+#if defined(KOKKOS_ARCH_VOLTA70) || defined(KOKKOS_ARCH_PASCAL60)
```

Comment:
What about `VOLTA72`, `TURING75`, `AMPERE80` and `AMPERE86`? We only want to set the number of teams for these two architectures or for all the architectures newer than Maxwell?

Reply:
Good point.  Please change to 
```suggestion
  #if defined(KOKKOS_ARCH_VOLTA) || defined(KOKKOS_ARCH_PASCAL) || \
      defined(KOKKOS_ARCH_MAXWELL) || defined(KOKKOS_ARCH_KEPLER)
```

Reply:
And add `#cmakedefine KOKKOS_ARCH_AMPERE` to `cmake/KokkosCore_config.h.in` it is missing

Reply:
Good Catch. We don't need KEPLER since the OpenMTarget directives are only supported from the generation of PASCAL GPUs.